Client/Client.py

```diff
@@ -362,14 +358,17 @@ def parseStreamOutput(output):
     speed = int(re.search(r'[0-9]+', speed).group()) if speed else None
     return (bench, speed)
 
-def computeSingleThreadedBenchmark(engine, outqueue):
+def computeSingleThreadedBenchmark(engine, outqueue, network):
 
     try:
 
         # Launch the engine and run a benchmark
         pathway = addExtension(os.path.join('Engines', engine).rstrip('/'))
+        cmdline = './{0} bench'
+        if network:
+            cmdline = cmdline + " option.EvalFile=" + os.path.join('Engines', network).rstrip('/')
         stdout, stderr = subprocess.Popen(
-            './{0} bench'.format(pathway).split(),
+            cmdline.format(pathway).split(),
```

Comment:
```python
 
        # Assume .exe extensions on Windows; Pass any Network files with the benchmark
        cmdline = './{0} bench'.format(addExtension(os.path.join('Engines', engine).rstrip('/')))
        if network: cmdline += " option.EvalFile=" + os.path.join('Engines', network).rstrip('/')

        # Launch the engine and run it's benchmark
        stdout, stderr = subprocess.Popen(cmdline, stdout=subprocess.PIPE ,stderr=subprocess.PIPE).communicate()
```

Reply:
Don't merge it yet, I am having some issues in Debian 10, narrowing it down.